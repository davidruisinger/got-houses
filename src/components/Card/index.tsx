import styled from '../_utils/styledComponents'

const Card = styled.div`
  border-radius: 4px;
  box-shadow: 2px 2px 10px rgba(0, 0, 0, 0.2);
  overflow: hidden;
  box-sizing: border-box;
`

export default Card
