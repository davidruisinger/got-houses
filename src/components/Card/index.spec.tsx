import 'jest-styled-components'
import React from 'react'
import testRenderer from 'react-test-renderer'
import Card from './index'

const tree = testRenderer.create(<Card />).toJSON()

describe('<Card />', () => {
  it('matches the snapshot', () => {
    expect(tree).toMatchSnapshot()
  })
})
