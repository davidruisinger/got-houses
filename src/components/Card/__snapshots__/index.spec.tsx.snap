// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> matches the snapshot 1`] = `
.c0 {
  border-radius: 4px;
  box-shadow: 2px 2px 10px rgba(0,0,0,0.2);
  overflow: hidden;
  box-sizing: border-box;
}

<div
  className="c0"
/>
`;
